### PR TITLE
Mypy

### PR DESCRIPTION
**Pull request by palango**
Add mypy analysis.

### CODE REVIEW

File: pathfinder/api/rest.py [hunk 1]
```diff
@@ -147,7 +147,7 @@ def put(self, token_network_address: str, channel_id: str):
         if channel_id_error is not None:
             return channel_id_error
 
-        channel_id = int(channel_id)
+        channel_id_casted = int(channel_id)
```
Comment (pcppcp) [inline pathfinder/api/rest.py]: you can use flask's request parser to validate the argument. And if I remember correctly, you can also force int type for part of the url.

https://github.com/raiden-network/microraiden/blob/master/microraiden/proxy/paywalled_proxy.py#L76

File: pathfinder/tests/fixtures/accounts.py [hunk 2]
```diff
@@ -14,6 +14,8 @@ def faucet_private_key(request) -> str:
     if is_hex(private_key):
         assert len(remove_0x_prefix(private_key)) == 64
         return private_key
+    else:
```
Comment (pcppcp) [inline pathfinder/tests/fixtures/accounts.py]: just raise NotImplementedError and let the user deal with it

File: pathfinder/api/rest.py (hunk 1 above)
Comment (palango) [reply pathfinder/api/rest.py]: Good point, but we decided to do that later and introduce our own converters then.

File: pathfinder/tests/fixtures/accounts.py (hunk 2 above)
Comment (palango) [reply pathfinder/tests/fixtures/accounts.py]: Fixed

**Pull request merged**